python_frank_energie/frank_energie.py

```
@@ -1927,7 +2300,7 @@ async def period_usage_and_costs(
 
         if isinstance(start_date, str):
             try:
-                start_date_obj = datetime.strptime(start_date, "%Y-%m-%d").date()
+                _start_date_obj = datetime.strptime(start_date, "%Y-%m-%d").date()
```

Flake8 (7.3.0)

[error] local variable '_start_date_obj' is assigned to but never used

(F841)